src/modules/protection/ContentProtectionModule.ts

```diff
@@ -740,8 +749,11 @@ export default class ContentProtectionModule implements ReaderModule {
     if (IS_DEV) {
       console.log("before print");
     }
-    this.delegate.headerMenu.style.display = "none";
-    this.delegate.mainElement.style.display = "none";
+
+    if (this.delegate && this.delegate.headerMenu) {
```

Comment:
I added this conditional because I was seeing this.delegate is undefined errors. It seemed harmless to add this check.